amf-client/shared/src/test/resources/upanddown/cycle/raml10/api-with-multi-line-schema/api.raml.expanded.jsonld

```diff
@@ -52,7 +52,7 @@
                 ],
                 "http://a.ml/vocabularies/document-source-maps#value": [
                   {
-                    "@value": "[(2,0)-(20,0)]"
+                    "@value": "[(2,0)-(19,5)]"
```

Comment:
some api's lost the final line break in their json schemas when cycling, don't really know why.

Reply:
Probable takes the Ymap as range instead of the YNode container. Is not relevant at this point.